specification/billing/resource-manager/Microsoft.Billing/preview/2020-11-01-preview/examples/PromotionActivation.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2019-11-01-preview",
+    "promotionId": "{promotionId}"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.Billing/promotions/ff0f1add-122a-43c1-a2ec-c2277781667d",
+        "type": "Microsoft.Billing/promotions",
+        "name": "ff",
```

Comment:
name value should match the value in id.

Reply:
ack.